File: Exemplos M10/App_IntentDigitalHUB_XamarinForms/Elgin_IntentDigitalHUB_XamarinForms-font/E1_Digital_Hub_Xamarin_Forms_M8/IntentServices/Termica/Corte.cs
namespace Xamarin_Forms_Intent_Digital_Hub.IntentServices.Termica
{
    class Corte : TermicaCommand
    {
        readonly int avanco;

        public Corte(int avanco) : base("Corte")
        {
            this.avanco = avanco;
        }

        protected override string FunctionParameters()
        {
            return "\"avanco\"" + ":" + avanco;
        }
    }
}

File: Exemplos SmartPOS/App_IntentDigitalHUB_ReactNative/Elgin_IntentDigitalHUB_SmartPOS_ReactNative-font/src/intentDigitalHubService/TERMICA/Commands/Corte.ts
import {TermicaCommand} from '../TermicaCommand';

export class Corte extends TermicaCommand {
  readonly avanco: number;

  constructor(avanco: number) {
    super('Corte');
    this.avanco = avanco;
  }

  functionParameters(): object {
    return {avanco: this.avanco};
  }
}
